### PR TITLE
Fix broken links.

### PR DESCRIPTION
**Pull request by jgerigmeyer**

### CODE REVIEW

File: content/blog/2013/susy-next.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ name.
 Please welcome the Susy Next team:
 
 - [Jina Bolton][] \[[@jina]\] is a Product Designer at Do, a regular
-  speaker at web conferences, co-creator of the [320 and Up]
+  speaker at web conferences, co-creator of the 320 and Up
```
Comment (jgerigmeyer) [inline content/blog/2013/susy-next.md:55]: @mirisuzanne A lot of these bios are now out-of-date, but maybe better to leave that in the context of when it was written?

File: content/blog/2013/susy-next.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ name.
 Please welcome the Susy Next team:
 
 - [Jina Bolton][] \[[@jina]\] is a Product Designer at Do, a regular
```
Comment (mirisuzanne) [inline content/blog/2013/susy-next.md:54]:
```suggestion
- [Jina][] \[[@jina]\] is a Product Designer at Do, a regular
```

File: content/blog/2013/susy-next.md [hunk 3]
```diff
@@ -86,17 +86,16 @@ help make it a community project. Stay tuned!
 
 [singularity]: https://singularity.gs/
 [salsa]: https://tsi.github.io/Salsa/
-[breakpoint]: http://breakpoint-sass.com
+[breakpoint]: http://breakpoint-sass.com/
 [susy]: /susy/
 [jina bolton]: https://github.com/jina/
```
Comment (mirisuzanne) [inline content/blog/2013/susy-next.md:91]:
```suggestion
[jina]: https://github.com/jina/
```

File: content/talks/dynamic-css.md [hunk 4]
```diff
@@ -56,7 +56,6 @@ events:
     slides: https://talks.oddbird.net/dynamic-css/vueconf19/
     video: https://www.vuemastery.com/conferences/vueconf-us-2019/dynamic-css-with-vue/
   - venue: Front Range Front-End
-    url: https://www.meetup.com/front-range-front-end/events/chtxtpyxpbcb/
```
Comment (jgerigmeyer) [inline content/talks/dynamic-css.md:59]: @mirisuzanne I guess removing these broken links means they now link to the page this is already on? Not a big deal. 🤷

Comment (mirisuzanne) [reply content/talks/dynamic-css.md:59]: @jgerigmeyer works for me

**Comment by jgerigmeyer** [comment]
@mirisuzanne I think this PR needs your 👍 before I can merge it?

**Review by mirisuzanne — APPROVED** [review]

**Pull request merged**